src/providers/ethereum/EthereumSwapProvider.js

```diff
@@ -83,16 +83,13 @@ export default class EthereumSwapProvider extends Provider {
 
   async claimSwap (initiationTxHash, recipientAddress, refundAddress, secret, expiration) {
     const initiationTransaction = await this.getMethod('getTransactionReceipt')(initiationTxHash)
-    const data = this.getRedeemSwapData(secret)
+    const data = padHexStart(secret, 64)
     return this.getMethod('sendTransaction')(initiationTransaction.contractAddress, 0, data)
   }
 
-  getRedeemSwapData (secret) {
-    return padHexStart(secret, 64)
-  }
-
-  getRefundSwapData () {
-    return ''
+  async refundSwap (initiationTxHash, recipientAddress, refundAddress, secretHash, expiration) {
```

Comment:
Should this be standardized as `secretHash` as an argument or `secret` as an argument?

Reply:
secrethash i believe as this is what the contract is created with. If secret was part of the interface, a party not knowing the secret couldn't get a refund which doesn't seem right